apps/web/components/Stats.tsx

```diff
@@ -0,0 +1,38 @@
+import { StatsInterface } from "../utils/getStatsAndContributors";
+
+interface StatsProps {
+  stats: StatsInterface;
+}
+
+export default function Stats({ stats }: StatsProps) {
+  return (
+    <section className="py-14">
+      <div className="container grid grid-cols-2 gap-y-9 gap-x-12 text-center sm:grid-cols-4">
+        <p className="space-y-1 xl:space-y-2">
+          <span className="block text-4xl font-bold xl:text-5xl">
+            {stats.templates}
+          </span>
+          <span className="block text-slate-400 xl:text-xl">Templates</span>
+        </p>
+        <p className="space-y-1 xl:space-y-2">
```

Comment:
One stat entry should be a component to avoid repetition. Doesn't need to be in a separate file name. Just create a new component below and use props to pass the dynamic parts.

Reply:
Ah, makes sense. thank you. I did it at first, the issue is the Object keys. I think I'll go with `{"Key With Spaces": "value"}` format, e.g. `{"GitHub Stars": 34}`

Reply:
```
interface Stat {
  title: string;
  value: string;
}

type Stats = Record<string, Stat>; // maps string key to Stat type

const stats: Stats = { 
  key: { title: "Commits", value: 20 }
 };
 ```
 
You can do something like that